libraries/cmake/source/librdkafka/CMakeLists.txt

```diff
@@ -103,41 +107,41 @@ function(generateLibrdkafka)
   
   if(DEFINED PLATFORM_WINDOWS)
     target_compile_definitions(thirdparty_librdkafka_c PRIVATE
-      LIBRDKAFKA_STATICLIB
       WITH_ZSTD
       WITH_ZSTD_STATIC
-      WITH_BUNDLED_ZSTD
```

Comment:
Very cool, I had been worried about this before.

Reply:
I'm not event sure where that comes from, because there's no trace of it being present/used in the librdkafka source code.